routes/articles.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const methodOverride = require('method-override');
+const articlesModule = require('../db/articles.js');
+
+let htmlContent = {};
+let error = { message : undefined };
+const router = express.Router();
+
+router.use(bodyParser.urlencoded({extended: true}));
```

Comment:
Since bodyParser and methodOverride is used in both the articles and products route, it's best to just move that into the server file.